packages/@aws-cdk/aws-appsync/test/appsync-apikey.test.ts

```diff
@@ -1,206 +0,0 @@
-import '@aws-cdk/assert/jest';
```

Comment:
confirming, since we are removing tests / removing this file that we haven't dropped any test cases right?

Reply:
yup! i just copy and pasta-d into appsync-auth